packages/core/src/middleware/koa-log.test.ts

```diff
@@ -60,33 +62,69 @@ describe('koaLog middleware', () => {
     });
   });
 
-  it('should insert log with failed result if next throws error', async () => {
-    const ctx: WithLogContext<ReturnType<typeof createContextWithRouteParameters>> = {
-      ...createContextWithRouteParameters({ headers: { 'user-agent': userAgent } }),
-      // Bypass middleware context type assert
-      addLogContext,
-      log,
-    };
-    ctx.request.ip = ip;
-
-    const error = new Error('next error');
-
-    const next = async () => {
-      ctx.log(type, payload);
-      throw error;
-    };
-    await expect(koaLog()(ctx, next)).rejects.toMatchError(error);
+  describe('should insert an error log with the error message when next() throws an error', () => {
+    it('should log with error message when next throws a normal Error', async () => {
+      const ctx: WithLogContext<ReturnType<typeof createContextWithRouteParameters>> = {
+        ...createContextWithRouteParameters({ headers: { 'user-agent': userAgent } }),
+        // Bypass middleware context type assert
+        addLogContext,
+        log,
+      };
+      ctx.request.ip = ip;
+
+      const message = 'normal error';
+      const error = new Error(message);
+
+      const next = async () => {
+        ctx.log(type, payload);
+        throw error;
+      };
+      await expect(koaLog()(ctx, next)).rejects.toMatchError(error);
+
+      expect(insertLogMock).toBeCalledWith({
+        id: nanoIdMock,
+        type,
+        payload: {
+          ...payload,
```

Comment:
This `payload` is a little bit confusing to me, was thinking this `payload` is `logger`'s property.
Maybe using `mockPayload` or something for better understanding.